### PR TITLE
refactor:Add autoformat before commit

### LINKED ISSUE #4: Rewrite formatter
The current formatter has enormous bug. We need to implement a new one

### PR DESCRIPTION
**Pull request by splasky**
This commit add clang format style from tangle-accerlator,
and autoformat the file which commited.

Closes #4

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -83,4 +84,6 @@ clean_mbedtls:
 clean_http_parser:
 	$(MAKE) -C $(HTTP_PARSER_PATH) clean
 
--include main.c.d
\ No newline at end of file
+pre-build:
+	git config core.hooksPath hooks
+-include main.c.d
```
Comment (jserv) [inline Makefile]: Use [String Substitution](https://www.gnu.org/software/make/manual/html_node/Text-Functions.html) to avoid hardcoded `main.c.d`.

File: Makefile [hunk 2]
```diff
@@ -86,4 +86,3 @@ clean_http_parser:
 
 pre-build:
 	git config core.hooksPath hooks
--include main.c.d
```
Comment (jserv) [inline Makefile]: Why did you remove dependency description?

Comment (splasky) [reply Makefile]: Not needed anymore.We just need object file to link.Previous dependency description is for debugging.
